src/main/java/no/digipost/concurrent/executor/DefaultExecutors.java

```diff
@@ -32,11 +33,15 @@
 
 
     public static ExecutorService fixedThreadPool(int threadAmount, String name) {
-        return Executors.newFixedThreadPool(threadAmount, threadNamingFactory(n -> name + "-" + n));
+        return Executors.newFixedThreadPool(threadAmount, threadNamingFactory(threadId -> name + "-" + threadId));
     }
 
     public static ExecutorService singleThreaded(String name) {
-        return Executors.newSingleThreadExecutor(threadNamingFactory(n -> name + "-" + n));
+        return Executors.newSingleThreadExecutor(threadNamingFactory(threadId -> name + "-" + threadId));
+    }
+
+    public static ScheduledExecutorService singleThreadedScheduled(String name){
```

Comment:
OK to rename this one to `scheduledSingleThreaded`? Can't exactly put my finger on what, it just sounds better to me 😛